.github/workflows/simulation-1.2.yml

```diff
@@ -284,13 +288,69 @@ jobs:
         name: cov-thread-1-2-backbone
         path: tmp/coverage.info
 
+  thread-border-routing:
+    runs-on: ubuntu-20.04
+    env:
+      REFERENCE_DEVICE: 1
+      VIRTUAL_TIME: 0
+      PACKET_VERIFICATION: 1
+      THREAD_VERSION: 1.2
```

Comment:
does this feature requires Thread version 1.2?

Reply:
No. But we are currently sharing the same OTBR docker facilities (designed for 1.2 testing) for the CI testing. It may be better to refactor the OTBR docker device to support both 1.2 and non-1.2 but may leave it from this PR.